CODE_OF_CONDUCT.md

```diff
@@ -2,7 +2,11 @@
 
 ⚠️ **All projects and contributors are governed by this document!** ⚠️
 
-The purpose of this guide is to ensure all contributors and projects within the [gliff.ai](https://gliff.ai) space have a unified Code of Conduct to create a positive and welcoming environment for all.
+The purpose of this guide is to outline [gliff.ai](https://gliff.ai)'s expectations of individuals involved in [gliff.ai](https://gliff.ai) activities. While incidents of unaccpetable behaviour are relatively infrequent, this document serves as a guide to form a clear set of standards to maintain a safe and welcoming space that avoids/prevents behaviours that prevent this, while outlining reporting and disciplinary processes.
+
+[gliff.ai](https://gliff.ai) employees a team of passionate and innovative backgrounds, with the outlook that our compnay, our products and our customers benefit from diveristy of backgrounds and perspectives. We prioritise creating a space which is a safe and welcoming environment for all, regardless of their personal identity or background, as anything other than thise can jeopardise the ability to work and with [gliff.ai](https://gliff.ai).
```

Comment:
We employee a team of backgrounds do we?

Reply:
Opps my bad! 🥴